drf_extra_fields/fields.py

```diff
@@ -240,3 +241,17 @@ class DateRangeField(RangeField):
     ModelSerializer.serializer_field_mapping[postgres_fields.DateRangeField] = DateRangeField
     ModelSerializer.serializer_field_mapping[postgres_fields.IntegerRangeField] = IntegerRangeField
     ModelSerializer.serializer_field_mapping[postgres_fields.FloatRangeField] = FloatRangeField
+
+
+class LowercaseEmailField(EmailField):
+    """
+    An enhancement over django-rest-framework's EmailField to allow
+    case-insensitive serialization and deserialization of e-mail addresses.
+    """
+    def to_internal_value(self, data):
+        data = super(LowercaseEmailField, self).to_internal_value(data)
+        return data.lower()
+
+    def to_representation(self, value):
+        value = super(LowercaseEmailField, self).to_representation(value)
+        return value.lower()
```

Comment:
`to_internal_value` mutates the data anyway. I think we don't need to override `to_representation`.

Reply:
If you decided to use this field in already started project (which has some data), it may have some emails with uppercase characters. You should update these emails or override `to_representation`. That's the reason I think.

Reply:
Don't we have to run a migration to add this field and change the current one? `to_internal_value ` method would run in this case. Am I mistaken?

Reply:
No, we don't have to run a migration. This is not a model field.

Reply:
I see, this is a serializer field. My bad. 🤦‍♂